account_move_batch_validate/account.py

```diff
@@ -25,12 +25,16 @@
 from openerp.osv import fields, orm
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job
-from openerp.addons.connector.session import ConnectorSession
-from openerp.addons.connector.queue.job import OpenERPJobStorage
-
 _logger = logging.getLogger(__name__)
 
+try:
+    from openerp.addons.connector.queue.job import job
+    from openerp.addons.connector.session import ConnectorSession
+    from openerp.addons.connector.queue.job import OpenERPJobStorage
+except ImportError:
+    _logger.debug('Can not `import connector`.')
```

Comment:
@yvaucher 
Imagine next case:
 - I use OCA/account-financial-tools
 - But I don't use `connector`
My custom runbot show a warning error that I need add `connector` repo and now I have build in yellow color.
This was discussed here: https://github.com/OCA/maintainer-tools/pull/92

Now, if you runbot
  - Use OCA/account-financial-tools and `connector` too
Then you will have odoo error of depends unnexistent, and all are happy.